test/integration/__fixtures__/transactions.ts

```diff
@@ -712,498 +710,6 @@ export const TransformTransactionTestcases: TransformTransactionBodyTestcase[] =
-                },
-                {
-                  type: RelayType.SINGLE_HOST_ADDRESS,
-                  port: 65535 as Port,
-                  ipv4: ipv4ToBuffer('127.0.0.255'),
-                  ipv6: null,
-                },
-              ],
-              poolMetadata: {
-                url: toMaxLenString('https://pool-metadata.vacuumlabs.com', 64),
-                metadataHash: toFixlenBuffer(
-                  'E318D62E3D5CC3CC23CA1123438E439D7AAC6C6C423320F670D159726AC9D11F',
-                  32,
-                ),
-              },
-            },
-          },
-          {
-            type: CertificateType.POOL_RETIREMENT,
-            poolKeyHash: toFixlenBuffer(
-              '4dfbc0559b2e1d6af62c447f0a0d6290a8b05e075ef08db38c1b81a8',
-              28,
-            ),
-            epoch: toUint(122814),
-          },
-        ],
-        withdrawals: [
-          {
-            rewardAccount: rewardAccount(
-              'stake1uykqf807mz7yrm00h0yrtjs2www2e9s4272sych0fz7078gjzd4uw',
-            ),
-            amount: toUint(12425116),
-          },
-          {
-            rewardAccount: rewardAccount(
-              'stake_test17pmqld54t5fp0v037gyd7m294v3unctmpjvy5tf6y2amlwqun8tdp',
-            ),
-            amount: toUint(129570000),
-          },
-        ],
-        update: undefined,
-        auxiliaryDataHash: toFixlenBuffer(
-          'AF0C57B5A82F10A5D87E3145612B7D9A812D1470A932D182E46C396231072941',
-          32,
-        ),
-        validityIntervalStart: toUint(124),
-        mint: [
-          {
-            policyId: toFixlenBuffer(
-              '4a7fc1c3490c30e23961d24f345c81f2dd1105643776098530f06cff',
-              28,
-            ),
-            tokens: [
-              {
-                assetName: Buffer.from('Token1') as MaxlenBuffer<32>,
-                amount: toInt(10000000),
-              },
-              {
-                assetName: Buffer.from(
-                  'TokenTokenTokenTokenTokenToken12',
-                ) as MaxlenBuffer<32>,
-                amount: toInt('9223372036854775807'),
-              },
-            ],
-          },
-        ],
-        scriptDataHash: undefined,
-        collateralInputs: undefined,
-        requiredSigners: undefined,
-        networkId: undefined,
-        collateralReturnOutput: undefined,
-        totalCollateral: undefined,
-        referenceInputs: undefined,
-      },
-      scriptWitnesses: addIndefiniteLengthFlag([
-        [
-          0,
-          Buffer.from(
-            '2C049DFED8BC41EDEFBBC835CA0A739CAC961557950262EF48BCFF1D',
-            'hex',
-          ),
-        ],
-        [
-          0,
-          Buffer.from(
-            '47BEBB24D73F10F876926192F9BCBA280727F51C128926A9EAA6FDAD',
-            'hex',
-          ),
-        ],
-      ]),
-      auxiliaryData: [new Map([[1234, { foo: 'bar' }]]), []],
-    },
-  },
-  {
-    /*
-        cardano-cli transaction build-raw \
```

Comment:
Can we move this comment down below to the corresponding tx test case? I believe the command I used was the same, except for using `--cddl-format`.